pkgs/tools/inputmethods/keym/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, libX11
+, libXtst
+, libXi
+}:
+
+stdenv.mkDerivation rec {
+  pname = "keym";
+  version = "unstable-2022-07-10";
+
+  src = fetchFromGitHub {
+    owner = "cwkx";
+    repo = pname;
+    rev = "67a6d39d45e17221353e06c39283b5636b46d25c";
+    sha256 = "sha256-v2eS7un2ABnpWBwuKq+0CeLX8ivtlNUjM2jRboKumOE=";
+  };
+
+  buildInputs = [ libX11 libXtst libXi ];
+
+  makeFlags = [
+    "PREFIX=$(out)"
+  ];
+
+  meta = with lib; {
+    homepage = "https://github.com/cwkx/keym";
+    description = "C tool to control mouse with keyboard for X11";
+    license = licenses.mit;
```

Comment:
Please add yourself to the maintainers and set `meta.mainProgram` if there's a main binary